packages-exp/auth-compat-exp/src/platform.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as impl from '@firebase/auth-exp/internal';
+import {
+  getUA,
+  isBrowserExtension,
+  isReactNative,
+  isNode,
+  isIE
+} from '@firebase/util';
+
+declare global {
+  interface Document {
+    documentMode?: number;
+  }
+}
+
+// TODO(avolkovi): Remove this after rebase and use the version in @firebase/util
```

Comment:
can you do this now?

Reply:
sure, i just need to rebase auth-next on master

Reply:
done